docs/diagrams/AddAnnotationSequenceDiagram.puml

```diff
@@ -0,0 +1,100 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":MarkParser" as MarkParser LOGIC_COLOR
+participant ":AddAnnotationCommandParser" as AddAnnotationCommandParser LOGIC_COLOR
+participant "a:AddAnnotationCommand" as AddAnnotationCommand LOGIC_COLOR
+participant ":OfflineCommandResult" as OfflineCommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "b:Bookmark" as Bookmark MODEL_COLOR
+participant "doc:OfflineDocument" as OfflineDocument MODEL_COLOR
+participant "an:Annotation" as Annotation MODEL_COLOR
+participant ":TrueParagraph" as TrueParagraph MODEL_COLOR
+end box
+
+[-> LogicManager : execute("annotate 1 p/p1 n/some remark")
+activate LogicManager
+
+LogicManager -> MarkParser : parseCommand("annotate 1 p/p1 n/some remark")
```

Comment:
can change all the long long text to multiple lines with `\n`. then the resulting picture won't be so long

Reply:
ahh ok thanks

Reply:
yeah it's ok, just make it as short as possible if not the diagram quite huge